src/design/plone/contenttypes/profiles/default/types/Event.xml

```diff
@@ -16,10 +15,8 @@
     <element value="collective.venue.behaviors.ILocation" remove="True"/>
     <element value="plone.eventlocation" remove="True"/>
     <element value="plone.eventattendees" remove="True"/>
-    <element value="design.plone.contenttypes.behavior.argomenti" remove="True"/>
-    <element value="design.plone.contenttypes.behavior.additional_help_infos" remove="True"/>
-    <element value="design.plone.contenttypes.behavior.luoghi_correlati" remove="True"/>
-    
-
+    <element value="plone.eventcontact" remove="True"/>
+    <element value="design.plone.contenttypes.behavior.argomenti"/>
```

Comment:
occhio all'ordinamento dentro ai fieldset che avevo messo in ordine anche quello su indicazione di giulia